File: AndroidStudioProjects/Ejercicio1Listener/app/src/main/java/com/example/ejercicio1listener/MainActivity.java
package com.example.ejercicio1listener;

import androidx.appcompat.app.AppCompatActivity;
import android.view.*;
import android.os.*;
import android.widget.*;

public class MainActivity extends AppCompatActivity {

    Button idBut;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        idBut = (Button)findViewById(R.id.button);
        idBut.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getBaseContext(), "Hello World", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: AndroidStudioProjects/Practica/app/src/main/java/com/example/practica/MainActivity.kt
package com.example.practica

import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    lateinit var idBut: Button // Usamos lateinit para inicializar más tarde

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        idBut = findViewById(R.id.button)

        idBut.setOnClickListener {
            Toast.makeText(this@MainActivity, "Prueba", Toast.LENGTH_SHORT).show()
        }
    }
}
